codespell_lib/data/dictionary.txt

```diff
@@ -52242,9 +52253,15 @@ timzone->timezone
 timzones->timezones
 tinterrupts->interrupts
 tiome->time, tome,
+tiomeout->timeout
+tiomeouts->timeouts
 tipe->type, tip,
 tipical->typical
 tipically->typically
+tiple->triple, tuple,
```

Comment:
Perhaps `tipple` too, although probably rare in written text.

Reply:
I couldn't find this typo.